src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -98,6 +112,99 @@ public Wallet(String chainId, ApiClient walletApiClient) {
         setApiClient(walletApiClient);
     }
 
+    /**
+     * Migrates Klaytn accounts to KAS Wallet API. <br>
+     * This method needs a RPC instance directing endpoint url of Node API. <br>
+     * Therefore, you must initialize the node api before using this function. <br>
+     *
+     * <pre>Example :
+     * {@code
+     * ArrayList<MigrationAccount> accountsToBeMigrated = new ArrayList<>();
+     * accountsToBeMigrated.add(new MigrationAccount.Builder()
+     *         .setAddress("0x{address}")
+     *         .setMigrationAccountKey(new SinglePrivateKey("0x{privateKey}"))
+     *         .build()
```

Comment:
```suggestion
     * accountsToBeMigrated.add(new MigrationAccount.Builder()
     *         .setAddress("0x{address}")
     *         .setMigrationAccountKey(new SinglePrivateKey("0x{privateKey}"))
     *         .build()
```

How about creating three different constructor instead of using builder? 
new MigrationAccount("0x{address}", "0x{private key}")
new MigrationAccount("0x{address}", ["0x{private key}", "0x{private key}"])
new MigrationAccount("0x{address}", [["0x{private key}", "0x{private key}"], ["0x{private key}", "0x{private key}"], ["0x{private key}", "0x{private key}"]])

Reply:
@jimni1222 
Thanks for your feedback. That would be a great idea 👍 
I updated my PR [here](https://github.com/ground-x/caver-java-ext-kas/pull/80/commits/9a2d250c43fda36db94e8d3a8ed42eb013e30b67).